examples/unconditional_image_generation/train_unconditional.py

```diff
@@ -596,6 +596,12 @@ def transform_images(examples):
                     accelerator.get_tracker("tensorboard").add_images(
                         "test_samples", images_processed.transpose(0, 3, 1, 2), epoch
                     )
+                elif args.logger == "wandb":
```

Comment:
Looks nice! Could you also add a quick code snippet in the beginning that verifies that wandb is correctly installed: https://github.com/huggingface/diffusers/blob/fd5c3c09afe29cbbde42cfa791ca598a07d94da9/examples/dreambooth/train_dreambooth_lora.py#L536

That'd be great!

Reply:
Thanks, @patrickvonplaten. I've added that and resolved the issues introduced with a merge clash on the main. Looks like we were both working on the DDPM steps! I've also reflected the changes in the onnxruntime script :)